docs/en/observability/apm/security/apm-agents/api-keys.asciidoc

```diff
@@ -78,9 +78,11 @@ The Applications UI has a built-in workflow that you can use to easily create an
 Only API keys created in the Applications UI will show up here.
 
 Using a superuser account, or a user with the role created in the previous step,
-open {kib} and navigate to **{observability}** → **Applications** → **Settings** → **Agent keys**.
+open {kib}. Find **Applications** in the main menu or use the {kibana-ref}/introduction.html#kibana-navigation-search[global search field]. Go to **Settings** → **Agent keys**.
```

Comment:
This doesn't actually work. You can find Applications in the main menu, but you need to search on `apm` to find APM apps. Surely this is a bug. How should we handle this?

![image](https://github.com/user-attachments/assets/b1d3baf8-c466-4470-9880-0df290b30f2a)